book/getting-started/run-a-wallaroo-application-vagrant.md

```diff
@@ -0,0 +1,142 @@
+# Run a Wallaroo Application in Vagrant
+
+In this section, we're going to run an example Wallaroo application. By the time you are finished, you'll have validated that your environment is set up and working correctly. If you haven't already completed the Vagrant setup [instructions](/book/getting-started/vagrant-setup.md), please do so before continuing. This guide assumes that the Wallaroo Vagrant Box is up and running.
+
+There are a few Wallaroo support applications that you'll be interacting with for the first time:
+
+- Our Metrics UI allows you to monitor the performance and health of your applications.
+- Giles receiver is designed to capture TCP output from Wallaroo applications.
+- Giles sender is used to send test data into Wallaroo applications over TCP.
+- Machida, our program for running Wallaroo Python applications.
+
+You're going to set up our "Celsius to Fahrenheit" example application. Giles sender will be used to pump data into the application. Giles receiver will receive the output, and our Metrics UI will be running so you can observe the overall performance.
+
+The Metrics UI process will be run in the background. The other three processes (receiver, sender, and Wallaroo) will run in the foreground.  We recommend that you run each process in a separate terminal.
+
+For each Terminal you're expected to setup, you'd have to run the following to access the Vagrant Box:
+
+```bash
+cd ~/wallaroo-tutorial/wallaroo/vagrant
+vagrant ssh
+```
+
+## Terminal 1, Start the Metrics UI
+
+To start the Metrics UI run:
```

Comment:
The SSH/login shell's PATH doesn't include the directory that contains the `metrics_reporter_ui` executable.

Reply:
@slfritchie addressed, you'd have to pull and re-provision to confirm

Reply:
Yup, it works now.